res/controllers/Denon-Prime-4-scripts.js

```diff
@@ -723,6 +726,14 @@ Prime4.PadSection = function(deck, offset) {
 
     this.offset = offset;
 
+    this.padModeSelectLeds = components.Component({
```

Comment:
```suggestion
    this.padModeSelectLeds = new components.Component({
```

Reply:
I tried adding `new`, it returns another error. Will post it here tomorrow

Reply:
Interesting. At least we're getting closer to the error.

Reply:
Here's the error with the latest mapping code. Changes have been pushed.
```
Uncaught exception: file:///home/whanake/.mixxx/controllers/Denon-Prime-4-scripts.js:731: TypeError: Property 'forEach' of object [object Object] is not a function
Backtrace: trigger@file:///home/whanake/.mixxx/controllers/Denon-Prime-4-scripts.js:731
@file:///home/whanake/.mixxx/controllers/Denon-Prime-4-scripts.js:764
@file:///home/whanake/.mixxx/controllers/Denon-Prime-4-scripts.js:768
@file:///home/whanake/.mixxx/controllers/Denon-Prime-4-scripts.js:512
@file:///home/whanake/.mixxx/controllers/Denon-Prime-4-scripts.js:267
```